### PR TITLE
Cleanup SpatialInertia documentation

### PR DESCRIPTION
**Pull request by traversaro**
@prashanthr05 Note that to build the documentation locally you need to run the `dox` target (i.e. `make dox`) and have doxygen installed (`apt install doxygen`).

### CODE REVIEW

File: src/core/include/iDynTree/Core/SpatialInertiaRaw.h [hunk 1]
```diff
@@ -31,9 +31,9 @@ namespace iDynTree
     class SpatialInertiaRaw
     {
     protected:
-        double m_mass; /** mass */
-        double m_mcom[3]; /** first moment of mass (i.e. mass * center of mass */
-        RotationalInertiaRaw m_rotInertia; /** rotational inertia */
+        double m_mass; /** Mass. */
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertiaRaw.h]: These aside comments are passed on as documentation to the next variable declaration. For instance, "Mass" is documented in m_mcom[3].

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 2]
```diff
@@ -84,14 +99,15 @@ namespace iDynTree
         Wrench biasWrench(const Twist & V) const;
 
         /**
-         * Return the derivative of the bias wrench with respect to the link twist.
+         * @brief Return the derivative of the bias wrench with respect to the link 6D velocity.
          *
-         * Defining \f$ M \f$ as this inertia, return the derivative
-         * with respect to V  of the bias wrench V.cross(M*V).
+         * Defining \f$ M \in \mathbb{R}^{6 \times 6} \f$ as this inertia, return the derivative
+         * with respect to \f$ \mathrm{v} = \begin{bmatrix} v \\ \omega \end{bmatrix} \in \mathbb{R}^6 \f$
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: In the documentation of method `biasWrench(const Twist& v)`, there is a missing identity matrix 1_3 in the 6d inertia matrix defintion.

Comment (prashanthr05) [reply src/core/include/iDynTree/Core/SpatialInertia.h]: The 6D link velocity is denoted with capital V in that method, which is different from the small v notation used in this method.

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 3]
```diff
@@ -121,12 +137,12 @@ namespace iDynTree
          * |:--------:|:-------:|:--------:|
          * |     0    | \f$ m \f$ | The mass of the rigid body |
          * |  2-4     |  \f$ m c \f$      | The first moment of mass of the rigid body |
-         * |  5-9     | \f$ \operatorname{vech}(I_o) \f$ | The 6 indipendent elements of the 3d inertia matrix (\f$ I_{xx} I_{xy} I_{xz} I_{yy} I_{yz} I_{zz} \f$). |
+         * |  5-9     | \f$ \mathop{vech}(I) \f$ | The 6 indipendent elements of the 3d inertia matrix, i.e. \f$ \begin{bmatrix} I_{xx} \\  I_{xy} \\  I_{xz} \\  I_{yy} \\  I_{yz} \\  I_{zz} \end{bmatrix} \f$ . |
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: mc elements 1-3; vech(I) elements 4-9 ?

Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: independent*

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 4]
```diff
@@ -158,14 +174,15 @@ namespace iDynTree
...
+         *   Y(\mathrm{v}) \in \mathbb{R}^{6 \times 10}
          * \f]
          * such that:
          * \f[
-         *   I v = Y(v)\alpha
+         *   M \mathrm{v} = Y(\mathrm{v}) \alpha
          * \f]
          *
-         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ I \f$ .
+         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ M \f$,
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: \mathbb{R}^{10}

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 5]
```diff
@@ -174,14 +191,15 @@ namespace iDynTree
...
+         *   Y(\mathrm{v},a) \in \mathbb{R}^{6 \times 10}
          * \f]
          * such that:
          * \f[
-         *   I a + v \overline{\times}^{*} I v   = Y(v,a)\alpha
+         *   M a + \mathrm{v} \overline{\times}^{*} M \mathrm{v}   = Y(\mathrm{v}, a)\alpha
          * \f]
          *
-         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ I \f$
+         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ M \f$,
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: \mathbb{R}^{10}

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 6]
```diff
@@ -195,17 +213,18 @@ namespace iDynTree
          *
          * Get the matrix
          * \f[
-         *   Y(v,v_r,a_r) \in \mathbb{R}^{6\times6}
+         *   Y(\mathrm{v},\mathrm{v}_r,a_r) \in \mathbb{R}^{6\times6}
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: Y(\mathrm{v},\mathrm{v}_r,a_r) \in \mathbb{R}^{6\times10} ?

File: src/core/include/iDynTree/Core/SpatialInertia.h [hunk 7]
```diff
@@ -195,17 +213,18 @@ namespace iDynTree
...
          * \f]
          * such that:
          * \f[
-         *   I a_r + (v \overline{\times}^{*} I - I v \times)  v_r   = Y(v,v_r,a_r)\alpha
+         *   M a_r + (\mathrm{v} \overline{\times}^{*} M - M \mathrm{v} \times) \mathrm{v}_r   = Y(\mathrm{v},\mathrm{v}_r,a_r)\alpha
          * \f]
          *
-         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ I \f$
+         * If \f$ \alpha \in \mathbb{R}^10 \f$ is the inertial parameters representation of \f$ M \f$, as returned by the
```
Comment (prashanthr05) [inline src/core/include/iDynTree/Core/SpatialInertia.h]: \mathbb{R}^{10}

File: src/core/include/iDynTree/Core/SpatialInertiaRaw.h (hunk 1 above)
Comment (diegoferigo) [reply src/core/include/iDynTree/Core/SpatialInertiaRaw.h]: Use `///< My comment`

File: src/core/include/iDynTree/Core/SpatialInertia.h (hunk 2 above)
Comment (traversaro) [reply src/core/include/iDynTree/Core/SpatialInertia.h]: I am  not getting this comment.

**Comment by traversaro** [comment]
@prashanthr05 I should have addressed all comments, except for https://github.com/robotology/idyntree/pull/435#discussion_r180929689 that I am not getting what I should fix.

File: src/core/include/iDynTree/Core/SpatialInertia.h (hunk 2 above)
Comment (prashanthr05) [reply src/core/include/iDynTree/Core/SpatialInertia.h]: Throughout the documentation $m$ is denoted as the mass of the rigid body. So when denoted in the 6D inertia matrix, shouldnt it wrtten as $m1_{3}$, where $1_{3}$ is a $3 \times 3$ identity matrix.

Correct me if I'm wrong.

**Comment by traversaro** [comment]
@prashanthr05 Done. Let me know if you can check this in vacation, otherwise I can merge as it is.

**Comment by prashanthr05** [comment]
@traversaro done. You can merge it :)

**Pull request merged**